docker/install_airflow.sh

```diff
@@ -11,6 +11,13 @@ PIPOPTS=""
 #export SLUGIFY_USES_TEXT_UNIDECODE=no
 export AIRFLOW_GPL_UNIDECODE=yes
 
+echo "** Installing numpy."
+pip $PIPOPTS install numpy
+echo "** Installing setuptools."
+pip $PIPOPTS install -U setuptools
+echo "** Installing psycopg2-binary"
+pip $PIPOPTS install psycopg2-binary
```

Comment:
Why are we installing libraries (via `yum`) and then using the pre-built psycopg2-binary package?

> You can also obtain a stand-alone package, not requiring a compiler or external libraries, by installing the psycopg2-binary package from PyPI https://pypi.org/project/psycopg2-binary/

Reply:
I tried the parcel generation without 'psycopg2-binary', the following error happened: 


    In file included from psycopg/psycopgmodule.c:27:
    ./psycopg/psycopg.h:30:2: error: #error "Psycopg requires PostgreSQL client library (libpq) >= 9.1"

    It appears you are missing some prerequisite to build the package from source.

    You may install a binary package by installing 'psycopg2-binary' from PyPI.
    If you want to install psycopg2 from source, please install the packages
    required for the build and try again.

    For further information please check the 'doc/src/install.rst' file (also at
    <http://initd.org/psycopg/docs/install.html>).

    error: command 'gcc' failed with exit status 1


Using both gcc and 'pyscopg2-binary' is solving the issue

Reply:
I do not understand why you are installing prerequisites for compiling the source and then you punt and install the binary package.  Pick one method.  Do not provide both solutions.